OracleGoldenGate/setupLinuxEnv.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# LICENSE CDDL 1.0 + GPL 2.0
+#
+# Copyright (c) 1982-2016 Oracle and/or its affiliates. All rights reserved.
+#
+# Since: December, 2016
+# Author: gerald.venzl@oracle.com
+# Description: Sets up the unix environment for DB installation.
+# 
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+# 
+# Setup filesystem and oracle user
+# Adjust file permissions, go to /opt/oracle as user 'oracle' to proceed with Oracle installation
+# ------------------------------------------------------------
+
+# Setup environment for GoldenGate.   Also install rlwrap, vim and Java 1.8 for other requirements.
+mkdir -p $OGG_INSTALL_DIR && \
+chown -R oracle:oinstall $INSTALL_DIR && \
+chown -R oracle:oinstall $OGG_INSTALL_DIR && \
+chmod ug+x $INSTALL_DIR && \
+chmod ug+x $OGG_INSTALL_DIR && \
+wget http://www.mirrorservice.org/sites/dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm && \
+rpm -Uvh epel-release-latest-7.noarch.rpm && \
+yum -y -q install rlwrap && \
+wget --no-cookies --no-check-certificate --header "Cookie: gpw_e24=http%3A%2F%2Fwww.oracle.com%2F; oraclelicense=accept-securebackup-cookie" "http://download.oracle.com/otn-pub/java/jdk/8u60-b27/jdk-8u60-linux-x64.rpm" && \
```

Comment:
Please do not bypass the licence acceptance mechanism to download Oracle JDK. You will need to document the JDK requirement and have users download it manually. You'll then need to copy the RPM Into the image via the Dockerfile during build.